src/main/java/org/mifos/connector/interledger/Application.java

```diff
@@ -0,0 +1,16 @@
+package org.mifos.connector.interledger;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cache.annotation.EnableCaching;
+
+@EnableCaching
+@SpringBootApplication
+@SuppressWarnings({ "HideUtilityClassConstructor" })
```

Comment:
Why suppress warning about Utility Class Constructor.

Reply:
Since this class is not implemented, Gradle build fails when run with the default Spring Application Constructor. So this is a placeholder for the complete code.